docs/layers/identity/design-decisions/decide-on-aws-cli-login.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Decide on AWS CLI Login"
+sidebar_label: "AWS CLI Login"
+description: Decide on a CLI tool that enables AWS login and credentials via SAML IDP for CLI and web console access.
+---
+import Intro from '@site/src/components/Intro';
+import KeyPoints from '@site/src/components/KeyPoints';
+import Note from '@site/src/components/Note';
+import TaskList from '@site/src/components/TaskList';
+
+<Intro>
+Decide on a CLI tool that enables AWS login and credentials via SAML IDP for CLI and web console access.
+</Intro>
+
+## Problem
+
+Users need some way to login into AWS when using the CLI or applying Terraform changes. We have AWS Identity Center or AWS SAML setup for an AWS organization, but we need a way to login to AWS locally.
+
+There are a number of tools that can help with this, but we need to decide on one.
+
+### Option 1: Use the AWS CLI
+
+First of all, we could use the AWS CLI to login to AWS. This is the most basic way to login to AWS, but it requires a lot of manual steps and is not very user-friendly.
+
+### Option 2: Use Leapp
+
+Alternatively we could use Leapp by Noovolari. This is a tool that allows you to login to AWS using SAML and then automatically generates temporary credentials for you to use in the CLI. Once setup, Leapp makes it very easy to login to AWS and use the CLI, assume roles across your accounts with Role Chaining pre-configured, and even launch directly into the AWS web console.
+
+Leapp historically has been recommended by Cloud Posse and a popular choice for this use case, but recently Noovolari has announced that they are shutting down the service. The [leapp](https://github.com/Noovolari/leapp) project itself will continue to be supported, but the paid option will be shut down. This presents bad optics for the project and may make it less attractive to use for future customers.
```

Comment:
```suggestion
> ![IMPORTANT]  
> Leapp has been a popular choice for this use case, but with Noovolari announcing the shutdown of their paid service, this could raise concerns about the long-term viability of the project. While the [Leapp](https://github.com/Noovolari/leapp) project will continue to be supported, the discontinuation of the paid option might make it less appealing to future users.
```